app/views/admin/questions/_question.html.slim

```diff
@@ -1,20 +1,21 @@
 -@questions.each do |question|
   tr.info
     td
-      =link_to admin_question_path(question)
+      = link_to admin_question_path(question)
         = question.id
     td
-      =link_to admin_topic_path(question.topic_id)
+      = link_to admin_topic_path(question.topic_id)
```

Comment:
@OrestF  so you just don't know how to use partials

partials shouldn't have any instance variables only local
change instance @question from partial body to just "question"
put object to partial locals
= render 'admin/questions/level', question: question